Fix number on the "Make X images" button

JeLuF opened the pull request:
With this change, the number of prompt variants is taken into account when computing the number of images that will be generated. 
`X = getPrompts().length * numOutputsTotalField.value`

Fixes https://discord.com/channels/1014774730907209781/1059237919178637484
https://discord.com/channels/1014774730907209781/1021695193499582494/1063402892058431518

![image](https://user-images.githubusercontent.com/5852422/212421764-04fa0d42-d6d8-4c86-b24e-acfabdd90145.png)

patriceac commented:
The formula used here is incorrect and underestimates the number of images that will be produced in some scenarios.
Repro steps:
Create a single-prompt task (e.g. 'an astronaut riding a horse') with number of images set to 6 total, 5 in parallel

Expected result: Make image button reads 'Make 10 Images'
Actual result: Make image button reads 'Make 6 Images'

When task is ran, 10 images are generated.

JeLuF commented:
Fixed the issue reported by @patriceac:
Create a single-prompt task (e.g. 'an astronaut riding a horse') with number of images set to 6 total, 5 in parallel

Previous result: "Make image" creates 10 images
Fixed result: "Make image" creates 6 images as requested (First job will create 5 images in parallel, the second job will only create 1 image)

cmdr2 commented:
Thanks!
